README.md

```diff
@@ -210,4 +233,14 @@ request
 |
 |--|---|----|------>
    1   3    10     #Replicas
-```
\ No newline at end of file
+```
+
+### Custom Resource Definition
+
+#### Types
+
+See [here](https://github.com/gardener/hvpa-controller/blob/a336e86111bbd5ec87f5a749003ba39b4735261e/api/v1alpha1/hvpa_types.go#L28-L305).
+
+#### Example
+
+See [here](https://github.com/gardener/hvpa-controller/blob/a698b08bc371525f666bb2f4766ccb9a403a0d4b/config/samples/autoscaling_v1alpha1_hvpa.yaml).
```

Comment:
s/a698b08bc371525f666bb2f4766ccb9a403a0d4b/master

Reply:
Thought `master` would be moving target. Hence, the permalink. WDYT?